src/pages/components/radio-button/code.mdx

```diff
@@ -1,25 +1,32 @@
 ---
 title: Radio button
 description: Radio buttons are used when two or more options are mutually exclusive, meaning the user must select only one option.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
-## Radio button
-
 <ComponentDemo
```

Comment:
Theming bug: label text not rendering correctly 

![image](https://user-images.githubusercontent.com/11670886/78407418-65cd0500-75ca-11ea-9f20-326352fc9977.png)